src/ansys/fluent/core/workflow.py

```diff
@@ -11,6 +12,27 @@
 from ansys.fluent.core.data_model_cache import DataModelCache
 from ansys.fluent.core.services.datamodel_se import PyCallableStateObject, PyMenuGeneric
 
+AttrMap = {}
+
+
+def camel_to_snake_case(input_arg: str) -> str:
+    """Convert camel case input string to snake case output string."""
+    return re.sub("((?<=[a-z0-9])[A-Z]|(?!^)[A-Z](?=[a-z]))", r"_\1", input_arg).lower()
+
+
+def populate_and_use_snake_case(attr: str, attr_dict):
```

Comment:
This function does not require a `dict` to be passed in. The caller should call `keys()` is the source object happens to be a `dict`.